python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -3397,6 +3397,14 @@ def test_df_caache(self):
         self.assert_eq(10, df.count())
         self.assertTrue(df.is_cached)
 
+    def test_collated_string(self):
```

Comment:
Shall we add the test to `pyspark.sql.tests.test_types.TypesTestsMixin`, so that `pyspark.sql.tests.connect.test_parity_types` can inherit automatically?

Reply:
@xinrong-meng Moved the test and added to `test_parity_types`, please check.

Reply:
Looks good, thank you!

Reply:
Just fyi we don't need to add the test to `test_parity_types`, it will inherit automatically.
Thanks @HyukjinKwon for the fix.